src/main/java/audio/harmonizer/Harmonizer.java

```diff
@@ -63,10 +68,14 @@ private int[] normalizePitch(double[] pitch) throws AppError {
 
     private double[] getRandomScale() {
         double[] scale = new double[] {
-                2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f,
-                2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f,
-                2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f,
-                2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f, 1/76f, 2/76f
+                2 / 76f, 1 / 76f, 2 / 76f, 1 / 76f, 2 / 76f, 2 / 76f, 1 / 76f, 2 / 76f, 1 / 76f, 2 / 76f, 1 / 76f,
```

Comment:
@ArtInLines Is this autoLint or do you actually prefer this? I think it's way more readable as compact fractions

Reply:
Looks like autoLint.. Also prefer the compact option. Would recall that change.